i18n/locales/ja.json

```
@@ -86,8 +76,16 @@
     "topNav": {
         "home": "ホーム",
         "about": "私たちについて",
+        "home": "ホーム",
         "submit": "医師情報の追加"
     },
+    "hamburgerMenu": {
+        "about": "私たちについて",
+        "home": "ホーム",
+        "contactus": "お問い合わせ",
```

The key for this string should be the same as what's referenced in the component and also en.json.
```suggestion
        "contact": "お問い合わせ",
```